common/language.cpp

```diff
@@ -41,26 +63,25 @@ const LanguageDescription g_languages[] = {
 	{    "fi", "fi_FI", "Finnish", FI_FIN },
 	{    "fr", "fr_FR", "French", FR_FRA },
 	{    "de", "de_DE", "German", DE_DEU },
-	{    "gr", "el_GR", "Greek", GR_GRE },
+	{    "el", "el_GR", "Greek", EL_GRC },
 	{    "he", "he_IL", "Hebrew", HE_ISR },
-	{    "hb", "he_IL", "Hebrew", HE_ISR }, // Deprecated
 	{    "hu", "hu_HU", "Hungarian", HU_HUN },
 	{    "it", "it_IT", "Italian", IT_ITA },
-	{    "jp", "ja_JP", "Japanese", JA_JPN },
+	{    "ja", "ja_JP", "Japanese", JA_JPN },
 	{    "kr", "ko_KR", "Korean", KO_KOR },
-	{    "lv", "lv_LV", "Latvian", LV_LAT },
+	{    "lv", "lv_LV", "Latvian", LV_LVA },
 	{    "nb", "nb_NO", "Norwegian Bokm\xE5l", NB_NOR },
-	{    "fa", "fa_IR", "Persian (Iran)", FA_IRN },
+	{    "fa", "fa_IR", "Persian", FA_IRN },
 	{    "pl", "pl_PL", "Polish", PL_POL },
 	{    "br", "pt_BR", "Portuguese (Brazil)", PT_BRA },
-	{    "pt", "pt_PT", "Portuguese (Portugal)", PT_POR },
+	{    "pt", "pt_PT", "Portuguese (Portugal)", PT_PRT },
 	{    "ru", "ru_RU", "Russian", RU_RUS },
+	{    "sr", "sr_RS", "Serbian", SR_SRB },
```

Comment:
Maybe we need "sr_RS@latin" here, since "sr_RS" seems to be cyrillic locale and the only currently supported game in Serbian is using latin script.

Reply:
The right way might be to include another parameter for script. We also don't differentiate between Traditional and Simplified Chinese either, but it looks like both are available in Wintermute.

Reply:
>We also don't differentiate between Traditional and Simplified Chinese either
In WME, "Chinese (China)" and "Chinese (Taiwan)" are used in "Simplified Chinese" and "Traditional Chinese" meaning respectfully. As far as I understand, those things are pretty much the same:
- Simplified Characters were introduced by China and are also used at Malaysia and Singapore, China is much bigger then Malaysia and Singapore in both area and population.
- Traditional Characters are still used at Taiwan, Hong Kong and Macau, Taiwan is much bigger then Hong Kong and Macau in both area and population.
I guess "Chinese (China)" and "Chinese (Taiwan)" can be renamed to "Chinese (Simplified)" and "Chinese (Traditional)".

ZH_ANY was added as a workaround for a WME game Reversion that contained 3 Chinese translation packages in a single game directory after an update: new Simplified and Traditional translations were added and old Chinese package was not deleted (and detected by detection.cpp). To fix this, we decided to mark all old Reversion Chinese games as ZH_ANY ("generic" Chinese) and use ZH_CHA/ZH_TWN for Simplified/Traditional.